website/src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import styles from "./LanguageSelector.module.scss";
+import { availableLanguages } from "@/i18n";
+import "flag-icons/css/flag-icons.min.css";
+
+interface FlagIconProps {
+  countryCode: string;
+}
+
+function FlagIcon({ countryCode = "" }: FlagIconProps) {
```

Comment:
Don't use flags for language selector. Use the language name directly in the language it designed for example : 
- English
- Français
- Deutsch 

The use will know how to read his language.

Reply:
We have the flag and the language name. We think having the flags make easier to read for the user.